src/common/sharly_chess_config.py

```diff
@@ -143,6 +143,12 @@ def console_show_level(self) -> bool:
     def experimental(self) -> bool:
         return self.stored_config.experimental
 
+    @property
+    def experimental_features(self) -> list[str]:
+        return [
+            _('Non-Papi tie-breaks'),
```

Comment:
All FIDE recognised tie-break (including ones that are not compatible with Papi)